docs/viewmodel-locator.md

```diff
@@ -1,19 +1,22 @@
 # Using the ViewModelLocator
+
 The `ViewModelLocator` is used to wire the `DataContext` of a view to an instance of a ViewModel using a standard naming convention.
 
-The Prism `ViewModelLocator` has an `AutoWireViewModel` attached property, that when set to `true` calls the `AutoWireViewModelChanged` method in the `ViewModelLocationProvider` class to resolve the ViewModel for the view, and then set the view’s data context to an instance of that ViewModel.
+The Prism `ViewModelLocator` has an `AutoWireViewModel` attached property, that when set to `true` calls the `AutoWireViewModelChanged` method in the `ViewModelLocationProvider` class to resolve the ViewModel for the view, and then set the view’s data context to an instance of that ViewModel. This behavior is on by default: if you don't want that for your view, you need to opt-out:
```

Comment:
For WPF, it's only on by default when using Region Navigation or the IDialogService. Instead of placing this as sentence, maybe use the `> [!Note]` syntax to highlight the behavior

Reply:
Updated. Thanks Brian!